### PR TITLE
feat: add toJSON method for good stringify

### PR DESCRIPTION
**Pull request by zxqfox**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -241,6 +241,15 @@ module.exports = class BemEntityName {
     }
 
     /**
+     * Return raw data for JSON.stringify.
+     *
+     * @returns {{block: string, elem: ?string, mod: ?{name: string, val: string|boolean}}}
```
Comment (blond) [inline index.js]: `val: string|boolean` -> `val: *`

File: index.js [hunk 2]
```diff
@@ -241,6 +241,15 @@ module.exports = class BemEntityName {
     }
 
     /**
+     * Return raw data for JSON.stringify.
```
Comment (blond) [inline index.js]: `JSON.stringify` ->

```
`JSON.stringify()`
```

File: index.js (hunk 1 above)
Comment (zxqfox) [inline index.js]: Почему так?

**Comment by zxqfox** [comment]
@blond Thank you for review. Why in Debugablity section?

**Pull request merged**